email_validator.gemspec

```diff
@@ -2,8 +2,8 @@ Gem::Specification.new do |s|
   s.name = 'email_validator'
   s.version = '2.2.4'
```

Comment:
[NOTICE]
You know, this version needs to bump up before gem push 👻

Reply:
I'm not sure why you are mentions this. Is there somewhere that I forgot up bump this? Did the automation fail somewhere? I don't understand why you are bringing this up.